### DISCUSSION TITLE
When will be github shops Xraytocat bundle available in unisex L size?

### DISCUSSION THREAD

**Discussion by techyminati** [community/community#22489] [New to GitHub]
<p>When will be github shops Xraytocat bundle available in unisex L size??</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>I’m also waiting :rofl:</p>
<p>if you are not doing it yet, you can make the notification via your email, at the left side of the screen, this button appears</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181071409-430dcfb4-b44e-477f-b500-b245cebbeb30.png" alt="image" data-base62-sha1="8NmrLdhEQG5Vy8f78KdoEpOE9UW" width="77" height="277"></p>
<p>then enter the necessary details</p>
<p>but as for me, I’m always visiting the site</p>
